CriteoAdViewer/Sources/AdViewer/AdConfig.swift

```diff
@@ -20,11 +20,13 @@
 struct AdConfig {
   let publisherId: String
   let adUnit: CRAdUnit
+  let externalAdUnitId: String
```

Comment:
Such property is only used for rewarded. Do you mind updating the methods generating GAM ad request object to use this property instead of the old one ? This should not change any behavior, but this might be wiser to do it now and don't forget about this particularity rather than having to debug this later asking why this doesn't work on native, banner, ...

Reply:
Updated usages in `GoogleAdViewBuilder` to make use of this variable instead of ad unit id for future proofing.